package.json

```diff
@@ -46,7 +46,7 @@
     "codegen:generateMergeRouters": "ts-node scripts/generateMergeRouters.ts",
     "codegen": "run-p codegen:*",
     "postinstall": "yarn codegen",
-    "build": "yarn codegen:entrypoints && rollup --config rollup.config.ts --configPlugin typescript",
+    "build": "yarn codegen:entrypoints && rollup --config rollup.config.ts --configPlugin rollup-plugin-swc3",
```

Comment:
I noticed this got rid of the rough cold starts when building. Nice!

Reply:
Yeah the `typescript` config plugin was type checking the whole codebase